src/mds/Beacon.cc

```diff
@@ -486,8 +486,10 @@ void Beacon::notify_health(MDSRank const *mds)
     health.metrics.push_back(m);
   }
 
-  // Report if we have significantly exceeded our cache size limit
-  if (mds->mdcache->cache_overfull()) {
+  // Report if we have significantly exceeded our cache size limit. Don't
+  // include reports from standby-replay MDSs, since replay of data by it
+  // can confuse users.
+  if (mds->mdcache->cache_overfull() && !mds->is_standby_replay() {
```

Comment:
Instead of eliminating this altogether, I think its still of use.
We could rather qualify what type of MDS this health metric is originating from.
e.g.
`"MDS (standby-replay) cache is too large ..."`
`"MDS (active) cache is too large ..."`

If not a health metric report, then at least a log to the log file would be prudent with the suggested changes.
We would definitely like to know where things have gone wrong.

Reply:
The ask is to avoid the s-r daemon send inode/stray counters as part of its cache warning message. The warning should still be emitted.

Reply:
Just leaving here for record, the changes has been made - https://github.com/ceph/ceph/pull/56065/files#diff-424cfd2c9d5fe51809ddcba91b609280e5c1010af79155196f7d543f1f3747cf